databricks_cli/configure/cli.py

```diff
@@ -30,6 +30,7 @@
 from databricks_cli.configure.config import profile_option, get_profile_from_context, debug_option
```

Comment:
Can't leave comment above: did you intentionally chmod `databricks_cli/cli.py`?

Reply:
I had to `chmod` it to execute it but I'll change it back for the commit